lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -711,6 +711,7 @@ def test_basic(self):
             "username": self.user.username,
             "created_at": "2015-05-19T00:00:00Z",
             "updated_at": "2015-05-19T00:00:00Z",
+            "read": True,
```

Comment:
Why did this line need to be added?  If I understand correctly, the behavior is that posting a response would bump the unread_comment_count to 1, but the new change to the forum service prevents that, so the unread_comment_count stays at 0 (below, on line 745), but I'm not clear why we need this here...